src/components/Base/Timeline/timeline.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
```

Comment:
somewhere the imported name is `className`, while in other place the name is `classnames`, should be consistent